packages/language-server/src/lib/documents/Document.ts

```diff
@@ -118,7 +118,7 @@ export abstract class Document implements TextDocument {
         return this.getText().split(/\r?\n/).length;
     }
 
-    languageId = '';
+    languageId = 'svelte';
```

Comment:
This is actually the base class of multiple language's sub-document. Could you also help checking does setting this makes any difference?

Reply:
Fixed.

I don't think it actually matters except in the case where a file extension can possibly hold multiple types of content and you need to differentiate. Still it seems like good practice to set it appropriately